generators/app/index.js

```diff
@@ -76,6 +83,12 @@ module.exports = class extends Generator {
             name: 'tablerows'
           }
         ]
+      },
+      {
+        type: 'confirm',
+        message: 'Do you require React and babel setup for sugary stuff?',
```

Comment:
okay, I like that this is optional. Can we think of a clearer way to phrase this, though? Perhaps something like "Initialise with React and Babel? This will allow you to use React and ECMAscript 2015 features" (assuming what I just said was true).

Reply:
We should also add this to the tutorial - https://github.com/intermine/bluegenes/blob/dev/tools/docs/tool-api-tutorial.md

Reply:
Okay, I'll update the text and add to tutorial.

Reply:
Hi, I've updated the text.